templates/Template k8s-etcd.xml

```diff
@@ -0,0 +1,100 @@
+                <discovery_rule>
+                    <uuid>b972f3ac7aa34d8cbebd5fbd8393b227</uuid>
+                    <name>k8s-etcd discovery</name>
+                    <key>k8s-etcd.discovery</key>
+                    <item_prototypes>
+                        <item_prototype>
+                            <uuid>f956d80f4b114f19b42c2aa34d50f879</uuid>
+                            <name>k8s-etcd etcd-health status {#ENDPOINT} in {#NAMESPACE} ({#CONTEXT})</name>
+                            <type>DEPENDENT</type>
+                            <key>k8s-etcd.values-health.health[{#CONTEXT},{#NAMESPACE},{#STATUS_NAME},{#ENDPOINT}]</key>
+                            <delay>0</delay>
+                            <trends>0</trends>
+                            <value_type>TEXT</value_type>
+                            <preprocessing>
+                                <step>
+                                    <type>JSONPATH</type>
+                                    <parameters>
+                                        <parameter>$.health</parameter>
+                                    </parameters>
+                                </step>
+                            </preprocessing>
+                            <master_item>
+                                <key>k8s-etcd.values-health[{#CONTEXT},{#NAMESPACE},{#STATUS_NAME},{#ENDPOINT}]</key>
+                            </master_item>
+                            <trigger_prototypes>
+                                <trigger_prototype>
+                                    <uuid>667b9f5808ac4802ad5ad5ae1cb65a8a</uuid>
+                                    <expression>last(/zabbix-agent-osso k8s-etcd/k8s-etcd.values-health.health[{#CONTEXT},{#NAMESPACE},{#STATUS_NAME},{#ENDPOINT}])&lt;&gt;&quot;true&quot;</expression>
+                                    <name>k8s-etcd etcd-health status not ok {#ENDPOINT} in {#NAMESPACE} ({#CONTEXT})</name>
+                                </trigger_prototype>
+                            </trigger_prototypes>
+                        </item_prototype>
+                        <item_prototype>
+                            <uuid>627ce33ff8aa465ca8a078764eb35bef</uuid>
+                            <name>k8s-etcd etcd-health {#ENDPOINT} in {#NAMESPACE} ({#CONTEXT})</name>
+                            <key>k8s-etcd.values-health[{#CONTEXT},{#NAMESPACE},{#STATUS_NAME},{#ENDPOINT}]</key>
+                            <history>0</history>
+                            <trends>0</trends>
+                            <value_type>TEXT</value_type>
+                        </item_prototype>
+                        <item_prototype>
+                            <uuid>e6e525d7e29045da94e1952b9ed592b1</uuid>
+                            <name>k8s-etcd dbSize {#ENDPOINT} in {#NAMESPACE} ({#CONTEXT})</name>
+                            <type>DEPENDENT</type>
+                            <key>k8s-etcd.values-status.dbSize[{#CONTEXT},{#NAMESPACE},{#STATUS_NAME},{#ENDPOINT}]</key>
```

Comment:
We kiezen hier graag zelf naming en niet met camelCase. Dus `dbsize` of `db-size`.

Reply:
dbsize van gemaakt